src/core/Akka.Persistence/Persistence.cs

```diff
@@ -156,17 +156,25 @@ private PluginHolder CreatePlugin(string configPath, Func<Type, string> dispatch
 
             var pluginConfig = _system.Settings.Config.GetConfig(configPath);
             var pluginTypeName = pluginConfig.GetString("class");
-            var pluginType = Type.GetType(pluginTypeName, true);
 
-            var shouldInjectConfig = pluginConfig.HasPath("inject-config") && pluginConfig.GetBoolean("inject-config");
-            var pluginDispatcherId = pluginConfig.HasPath("plugin-dispatcher")
-                ? pluginConfig.GetString("plugin-dispatcher")
-                : dispatcherSelector(pluginType);
-            var pluginActorArgs = shouldInjectConfig ? new object[] { pluginConfig } : null;
-            var pluginActorProps = new Props(pluginType, pluginActorArgs).WithDispatcher(pluginDispatcherId);
-
-            var pluginRef = _system.SystemActorOf(pluginActorProps, configPath);
-            return new PluginHolder(pluginRef, CreateAdapters(configPath));
+            try
+            {
+                var pluginType = Type.GetType(pluginTypeName, true);
+                var shouldInjectConfig = pluginConfig.HasPath("inject-config") && pluginConfig.GetBoolean("inject-config");
+                var pluginDispatcherId = pluginConfig.HasPath("plugin-dispatcher")
+                    ? pluginConfig.GetString("plugin-dispatcher")
+                    : dispatcherSelector(pluginType);
+                var pluginActorArgs = shouldInjectConfig ? new object[] { pluginConfig } : null;
+                var pluginActorProps = new Props(pluginType, pluginActorArgs).WithDispatcher(pluginDispatcherId);
```

Comment:
cc @Aaronontheweb @Horusiath should pluginType be checked for ActorBase compat here?